test/gtest/uct/uct_p2p_test.cc

```diff
@@ -15,6 +15,8 @@ int             uct_p2p_test::log_data_count = 0;
 ucs_log_level_t uct_p2p_test::orig_log_level;
 
 
+std::string const uct_p2p_test::mem_types[] = {"host", "cuda"};
```

Comment:
maybe we need to add this to UCT base? since the enum is defined in UCT
eg `const char *uct_mem_type_names[]`